libraries/AP_GPS/AP_GPS.h

```diff
@@ -34,8 +34,6 @@
 // the number of GPS leap seconds
 #define GPS_LEAPSECONDS_MILLIS 18000ULL
 
-#define UNIX_OFFSET_MSEC (17000ULL * 86400ULL + 52 * 10 * MSEC_PER_WEEK - GPS_LEAPSECONDS_MILLIS)
```

Comment:
@OXINARF .. My confirmation was missing. I will confirm the value in pixhawk from now on.

> I'm not aware of anywhere in the C++ spec that says it will cast to the largest type in the full math expression; it will cast to the largest of the two operands in an math operation, not the entire math expression

example
https://en.wikipedia.org/wiki/Type_conversion
this writed
"In a mixed-type expression, data of one or more subtypes can be converted to a supertype as needed at runtime so that the program will run correctly. "